system-tests/e2e-transfer-test/backend-service/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+plugins {
+    `java-library`
+}
+
+dependencies {
+    api(project(":extensions:http"))
+    api(project(":core:boot"))
+    api(project(":core:base"))
+
+    api("jakarta.ws.rs:jakarta.ws.rs-api:3.0.0")
```

Comment:
Why are you using a fixed jakarta version? Everywhere else we are using the version number from the root project (e.g. https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/3b07d9c7fe49ebe50b082082858479e5a6465c85/samples/02-health-endpoint/build.gradle.kts#L33)

Reply:
no reasons, just inherited it from another project